modules/nf-neuro/registration/antsapplytransforms/meta.yml

```diff
@@ -63,28 +58,42 @@ args:
   - dimensionality:
       type: number
       description: |
-        Dimensionality of the input image.
+        Dimensionality of input images.
         e.g. `2` for 2D images, `3` for 3D images.
   - image_type:
       type: integer
       description: |
-        Type of the input image.
+        Type of input images.
         e.g. `0` for scalar images, `1` for vector images, `2` for tensor images, `3` for time series, `4` for multichannel, and `5` for five-dimensional image.
   - interpolation:
       type: string
       description: |
         Interpolation method to use for the registration.
         e.g. `Linear`, `NearestNeighbor`, `BSpline`, `Gaussian`, etc.
-      choices: |
-        ["Linear", "NearestNeighbor", "BSpline", "Gaussian", "Label", "MultiLabel", "CosineWindowedSinc", "WelchWindowedSinc",
-        "HammingWindowedSinc", "LanczosWindowedSinc", "GenericLabel"]
+      choices:
+        - Linear
+        - NearestNeighbor
+        - BSpline
+        - Gaussian
+        - Label
```

Comment:
there is no "Label" option

Reply:
Updated following antsApplyTransforms CLI, to come soon !